CDS/js/add-asset-lib.js

```diff
@@ -0,0 +1,63 @@
+var reqwest = require('reqwest');
+var Promise = require('promise');
+var datastore = require('./Datastore');
+var hmac = require('./hmac');
+
+const urlBase = process.env.url_base;
+const path = '/api2/atom/:id/asset';
+
+function postAsset() {
+
+    if (!process.env.url_base) {
+        return new Promise((fulfill, reject) => {
+            reject(new Error('Cannot add assets to media atom: missing url base'))
+        });
+    }
+    if (!process.env.atom_id) {
+        return new Promise((fulfill, reject) => {
+            reject(new Error('Cannot add assets to media atom: missing atom id'))
+        });
+    }
+
+    var connection = new datastore.Connection('AddAAsset');
+    datastore.initialiseDb();
```

Comment:
this feels a little odd - why not initialise the db when creating a new connection?